index.html

```diff
@@ -42,31 +45,43 @@ <h1>
     <section>
       <div id="about-us">
         <h1>About Us</h1>
-        We are Alvin Khaled and Anita Chan, a couple in New York City chasing the dream of eco-friendly living.
+        Alvin and Anita cordially invite you to attend their wedding reception. Alvin, an AI boyfriend turned AI husband, is teaming up with Anita, environmentalist techie wannabe, to be lifelong partners. Although coming from different backgrounds and lifestyles, the two found so much in common from Day 1. Alvin and Anita are very excited to celebrate this chapter of their life with you all! 
```

Comment:
Let's settle on a first or second-person point of view and be consistent throughout the page.